src/Collections/Sequences/LittleEndianNatConversions.dfy

```diff
@@ -58,8 +58,11 @@ abstract module LittleEndianNatConversions {
     ensures E > 0
   {
     LemmaDivBasicsAuto();
+    assert forall x :: x != 0 ==> 0 / x == 0;
     LemmaPowMultipliesAuto();
     LemmaFundamentalDivMod(Large.BITS(), Small.BITS());
+    assert Large.BITS() == Small.BITS() * (Large.BITS() / Small.BITS()) + (Large.BITS() % Small.BITS());
+    assert (Large.BITS() / Small.BITS()) > 0;
```

Comment:
Should these be lemma calls?

Reply:
They're specializations of the more general `ensures` clauses of `LemmaFundamentalDivMod`, so the only way I can think of to do that would be to have another fully-included but less general lemma to call here, which seems unsatisfying.